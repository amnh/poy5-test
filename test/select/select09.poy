(* 'select' best trees *)
set(seed:1)
read ("1.fas", "30.ss", aminoacids:("40.aa"))
build(4)
report(treestats)
select(best:2)
report(treestats)