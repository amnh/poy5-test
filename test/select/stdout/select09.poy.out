Trees Found:
                                            
              Tree length    Number of hits
              2817.          1
              2818.          1
              2841.          1
              2849.          1
Trees Found:
                                            
              Tree length    Number of hits
              2817.          1
              2818.          1
